dialect/sql/schema/schema.go

```diff
@@ -170,6 +170,9 @@ func (c *Column) ConvertibleTo(d *Column) bool {
 	case c.UintType() && d.IntType():
 		// uintX can not be converted to intY, when X > Y.
 		return c.Type-field.TypeUint8 <= d.Type-field.TypeInt8
+	// mariadb json == longtext
+	case c.Type == field.TypeString && d.Type == field.TypeJSON:
+		return true
```

Comment:
I prefer the dialect-specific logic to be in its own file (`mysql.go`). Also, IMO it should be handled when reading the column information.

We can detect if this is a MariaDB database by parsing the `version`.

Reply:
done

Reply:
```
&{content string longtext  2147483647  false false false <nil> [] []}
&{images string longtext  2147483647  false false false <nil> [] []}
````

it seems not works well.
If content defiened to `longtext` and images to `json`, we got both longtext.
so the last commit which can due to normal `longtext` field to `json`.
this could be a bug.